egs/arctic/vc/conf/train_pytorch_transformer.nopt.n80.single.yaml

```diff
@@ -0,0 +1 @@
+tuning/train_pytorch_transformer.nopt.v1.1.n80.single.yaml
```

Comment:
How about making only the link to the best config? (We can keep `conf/tuning` as is)

Reply:
Do you mean to remove the `nopt` ones (without pre-training)? My original intention was to let users train models even if they don't want to use the pre-training technique (so they can fully reproduce the results in the paper). If you feel we can keep the best setting (i.e. TTS pre-training with the complete 1-hour data) I can do it.

Reply:
Thank you for your explanation.
Then, we can keep the links to `nopt` and `tts_pt` config.
I think `n80` is optional but if you want to keep as is, it is OK.